frontendActions/build.gradle

```diff
@@ -16,3 +16,9 @@ dependencies {
 
 addIntellijToCompileClasspath(withGit4Idea: true)
 applySubtypingChecker()
+
+apply plugin: 'kotlin'
```

Comment:
`apply plugin: 'org.jetbrains.kotlin.jvm'` for consistency with declaration in top-level build.gradle